src/jqLite.js

```diff
@@ -519,29 +521,34 @@ function jqLiteDocumentLoaded(action, win) {
   }
 }
 
+function jqLiteReady(fn) {
+  var fired = false;
+
+  function trigger() {
+    if (fired) return;
+    fired = true;
```

Comment:
Since we are wiring up both `DOMContentLoaded` and `load` events, is it not possible that they will both be triggered even if we remove the event handlers from inside the `trigger` function?
In other words, if the browser queues up both handlers before either handler has run, then removing the handlers inside one of the handlers would not prevent the other from running.